test/games/common/test_dotnet.py

```diff
@@ -0,0 +1,5 @@
+from randovania.games.common.dotnet import is_dotnet_set_up
+
+
+def test_dotnet_setup():
+    is_dotnet_set_up()
```

Comment:
you're regressing in functionality.
You should essentially have 3 cases:
- return code returns 0
- return code returns 1
- dotnet cannot be found

Reply:
in that case is_dotnet_set_up() should return the error code?

Reply:
no. you just mock the result, similar to what the previous tests did

Reply:
pushed changes for this